server/update_branch.go

```diff
@@ -2,60 +2,84 @@ package server
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"net/http"
 	"strings"
 
-	"github.com/mattermost/mattermost-server/v5/mlog"
+	"github.com/mattermost/mattermost-mattermod/model"
 
 	"github.com/google/go-github/v32/github"
 )
 
-func (s *Server) handleUpdateBranch(ctx context.Context, eventIssueComment IssueComment) {
+var (
+	ErrCommenterPermission    error = errors.New("commenter does not have permissions")
+	ErrOrganizationPermission error = errors.New("we don't have permissions")
+	ErrUpdatePullRequest      error = errors.New("could not update pull request")
```

Comment:
I don't think we really need to specify the `error` type. Directly assigning is also fine. Unless you explicitly meant to do that?

Reply:
Actually these can be removed for now. They left from my complete isolation trial :)